src/webrtc/bitrate.ts

```diff
@@ -1,3 +1,78 @@
-import { TrackEncoding } from './types';
+import {
+  type BandwidthLimit,
+  RemoteTrackId,
+  TrackContext,
+  TrackEncoding,
+} from './types';
+import { TrackContextImpl } from './internal';
+import { findSender } from './peerConnectionUtils';
 
-export type Bitrates = Record<TrackEncoding, number> | number;
+export type Bitrate = number;
+export type Bitrates = Record<TrackEncoding, Bitrate> | Bitrate;
+
+export const defaultBitrates = {
+  audio: 50_000 as Bitrate,
```

Comment:
Is there any explanation somewhere on why these numbers?

Reply:
I've gathered some information, and added a comment: `// The suggested bitrate values are based on our internal tests.`